Gemfile

```diff
@@ -4,6 +4,7 @@ gem "pg", "~> 0.18"
 gem "puma", "~> 3.0"
 gem "sass-rails", "~> 5.0"
 gem "coffee-rails", "~> 4.2"
+gem "uglifier", "~> 3.0.2"
```

Comment:
I don't know, the push was failing for not having that gem in here (It's a default rails gem which I removed).